[FEATURE] Restructure Main Menu (#160)
TYPO3-Documentation/DocsTypo3Org-Homepage: Documentation/Home/ExtensionManuals.rst

.. container:: row m-0 p-0

   .. container:: col-md-6 pl-0 pr-3 py-3 m-0

      .. container:: card px-0 h-100

         .. rst-class:: card-header

            .. rubric:: Getting started with extension development

         .. container:: card-body

            *  :ref:`Introduction to extensions in TYPO3
               <t3coreapi:extension-architecture-introduction>`

            *  Extension :ref:`t3coreapi:extension-files-locations`

            *  :ref:`t3coreapi:extension-naming` and :ref:`t3coreapi:cgl`

   .. container:: col-md-6 pl-0 pr-3 py-3 m-0

      .. container:: card px-0 h-100

         .. rst-class:: card-header

            .. rubric:: TYPO3 Explained - API Quicklinks

         .. container:: card-body

            *  :ref:`QueryBuilder <t3coreapi:database-query-builder>` based on Doctrine
            *  :ref:`t3coreapi:DependencyInjection`
            *  :ref:`t3coreapi:request-handling`

Extension Manuals
==================

.. container:: row m-0 p-0

   .. container:: col-md-6 pl-0 pr-3 py-3 m-0

      .. container:: card px-0 h-100

         .. rst-class:: card-header

            .. rubric:: System Extensions

         .. container:: card-body

>             Here you can find documentation of system extensions and third party extensions,
>             if available.

Reviewer: ```suggestion
            Here you can find documentation for system extensions, as well as third party extensions,
            if available.
```

Reviewer: Oh woops - you mean we're still missing docs for some system extensions? Ignore my suggestion then.

Author: Yes, sadly we do :/
